PocketKit/Sources/Sync/PersistentContainer.swift

```diff
@@ -7,7 +7,7 @@ import CoreData
 public class PersistentContainer: NSPersistentContainer {
     public lazy var rootSpace = { Space(backgroundContext: backgroundContext, viewContext: viewContext) }()
 
-    private lazy var backgroundContext = {
+    internal lazy var backgroundContext = {
```

Comment:
`internal` is implicit; we don't need this. i think we investigated a swift lint rule for this, but i don't know where we landed.